skeletons/INTEGER.h

```diff
@@ -43,6 +43,8 @@ xer_type_encoder_f INTEGER_encode_xer;
 per_type_decoder_f INTEGER_decode_uper;
 per_type_encoder_f INTEGER_encode_uper;
 
+#define INTEGER_constraint asn_generic_no_constraint
```

Comment:
Take another expression in `rfc3280-PKIX1Explicit88.asn1` as an example :

```
ExtensionAttribute ::=  SEQUENCE {
   extension-attribute-type [0] IMPLICIT INTEGER
                   (0..ub-extension-attributes),
   extension-attribute-value [1]
                   ANY DEFINED BY extension-attribute-type }
```
`extension-attribute-type` has a range constraint and my PR indeed generate the corresponding `memb_extension_attribute_type_constraint_1` as original code did.

This is another evidence that my PR work the same way as original code.